pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -84,6 +87,33 @@ struct bpf_map_def SEC("maps/udp_recv_sock") udp_recv_sock = {
     .namespace = "",
 };
 
+#if ROUTE_LOOKUP_USE_EBPF
+/*
+ * This map is used to store the parameters for a call to ip_route_output_flow
+ * to match them later to a kretprobe
+ * key is the pid/tid
+ * value is ip_route_flow_t
+*/
+struct bpf_map_def SEC("maps/ip_route_output_flows") ip_route_output_flows = {
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(__u64),
+    .value_size = sizeof(ip_route_flow_t),
+    .max_entries = 1024,
+    .pinning = 0,
+    .namespace = "",
+};
+
+struct bpf_map_def SEC("maps/dest_gateways") dest_gateways = {
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(dest_tuple_t),
```

Comment:
dest_tuple_t is a confusing name, and everywhere else in the tracer code a 'tuple' is a full 7-tuple (5-tuple + pid + namespace).

Is there a better name we can think of for this? One suggestion is ip_tuple_and_ns_t, but I'm not thrilled with how verbose that is.